tools/src/test/java/org/apache/kafka/tools/consumer/group/ShareGroupCommandTest.java

```diff
@@ -588,6 +591,94 @@ public void testGroupStatesFromString() {
         assertThrows(IllegalArgumentException.class, () -> ShareGroupCommand.groupStatesFromString("   ,   ,"));
     }
 
+    @Test
+    public void testDeleteShareGroupsSuccess() {
+        String firstGroup = "first-group";
+        String secondGroup = "second-group";
+        String bootstrapServer = "localhost:9092";
+
+        String[] cgcArgs = new String[]{"--bootstrap-server", bootstrapServer, "--delete"};
```

Comment:
I would have expected the command line parsing to fail because `cgcArgs` doesn't include either `--group` or `--all-groups`. Similar comment for the other tests.